templates/index.html

```diff
@@ -5,9 +5,10 @@
         <h1>Welcome to eFolder Express!</h1>
         <p class="lead">
             <form action="/efolder-express/download/" method="post">
-                <label for="file-number" class="sr-only">File number</label>
-                <input id="file-number" type="text" class="form-control" placeholder="File number (e.g. 123456789)" aria-describedby="basic-addon2" name="file_number" autofocus="autofocus">
-                <br />
+                <div class="form-group">
+                    <label for="file-number">File number (e.g. 123456789):</label>
+                    <input id="file-number" type="text" class="form-control" name="file_number" autofocus="autofocus" placeholder="File number">
+                </div>
```

Comment:
Since you already us a label for this field above ... the "File number" placeholder for non IE browsers is a little redundant. 

Generally, if I'm using both a label and ghost-text in form fields, the label will be a description of what the field should be (like "Name") and the ghost text will be an example of what the content should look like (like "Ada Lovelace").